_posts/2018-11-02_bazelcon-2018-recap.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: posts
+title: "BazelCon 2018 Recap: presentation videos are up!"
+authors:
+  - davidstanke
+---
+
+Thank you to everyone who made BazelCon 2018 a success! Over 300 attendees, representing over 100 organizations, spent two packed days sharing experiences and envisioning the future of Bazel. 80+ users took advantage of Office Hours to get hands-on help with Bazel and [Remote Build Execution](https://blog.bazel.build/2018/10/05/remote-build-execution.html). And 40+ people received an immersive introduction to Bazel at Boot Camp.
+
+Couldn't make it? Want to share with colleagues? Or simply want a replay? Here are [all of the presentation videos](https://www.youtube.com/playlist?list=PLxNYxgaZ8Rsd3Nmvl1W1B4I6nK1674ezp):
+
+## BazelCon 2018 Videos
+
+**[Keynote: Testing Yields Velocity](https://www.youtube.com/watch?v=DVYRg6b2UBo&list=PLxNYxgaZ8Rsd3Nmvl1W1B4I6nK1674ezp&t=0s&index=2)** *Melody Meckfessel*
```

Comment:
Do you think it would be better to add the embedded video player here?

Reply:
Hmm... I'm skeptical of embedding the player... these pages go through a lot of pre-processing (which is continuing to evolve). I think it's fragile and likely to break. Also, if anyone's using a feed reader etc. So I err on the side of simplicity. Happy to discuss if you think I'm super wrong.

Reply:
That's fair, I was just thinking out loud :)